mockgcp/mockkms/cryptokeyversion.go

```diff
@@ -123,17 +123,11 @@ func (r *kmsServer) CreateCryptoKeyVersion(ctx context.Context, req *pb.CreateCr
 	now := time.Now()
 
 	var obj *pb.CryptoKeyVersion
-	if req.GetCryptoKeyVersion() == nil {
```

Comment:
`CryptoKeyVersion` is required in the CreateKeyVersion request. The code should not hit this line.